### DISCUSSION TITLE
AMS reader connected to a "Sagemcom T210-D" from Netz-noe

### DISCUSSION THREAD

**Discussion by Noschvie** [UtilitechAS/amsreader-firmware#835] [Q&A]
Hello
have the device "µHAN mosquito by dbeinder" (discussion [480](https://github.com/UtilitechAS/amsreader-firmware/discussions/480), ESP32c3) running and connected to a Sagemcom T210-D.
Using version 2.2.28 the HAN connection is ok, but with version 2.3.8 the HAN connection seems to be broken.
And I'm missing the Debug section in 2.3.8
How can the debugging be done?
Thanks

**Comment by ArnieO** [comment]
There are currently some issues with compiled code for ESP32-C3 (memory issues). We know how to remedy it, and it will be fixed - but it is not a priority - as we use ESP32-S2 in our designs. 

One consequence is that debugging is disabled in firmware v2.3.x for ESP32-C3.

I recommend you to stay on v2.2.28 until you see in the release notes that the ESP32-C3 issue has been solved.
